cli/endpoints/amlarc/managed/canary-declarative-flow/1-create-endpoint-with-blue.yaml

```diff
@@ -0,0 +1,45 @@
+$schema: https://azuremlsdk2.blob.core.windows.net/latest/onlineEndpoint.schema.json
+name: my-endpoint
+type: online
+auth_mode: aml_token
+traffic:
+  blue: 100
+identity:
+  type: system_assigned
+target: azureml:/subscriptions/{{subscriptionId}}/resourceGroups/{{resourceGroup}}/providers/Microsoft.MachineLearningServices/workspaces/{{workspaceName}}/computes/{{computeName}}
+tags:
+  tag1: first tag
+  tag2: second tag
+properties:
+  prop1: first prop
+  prop2: second prop
+
+deployments:
+  #blue deployment
+  - name: blue    
+    model:
+      name: model-1
+      version: 1
+      local_path: ../../model-1/model/sklearn_regression_model.pkl
+    code_configuration:
+      code: 
+        local_path: ../../model-1/onlinescoring/
+      scoring_script: score.py
+    environment:
+      name: env-model1
+      version: 1      
+      path: .
+      conda_file: file:../../model-1/environment/conda.yml
+      docker:
+        image: mcr.microsoft.com/azureml/intelmpi2018.3-ubuntu16.04:20210301.v1
```

Comment:
This needs to be updated later when the new base image is released.
This base image doesn't support swagger generation.

Reply:
Will update this when new image is released